docs/developer/shipping/shipping-method.mdx

```diff
@@ -0,0 +1,424 @@
+
+```graphql {2-7}
+mutation {
+  shippingPriceUpdate(
+    id: "U2hpcHBpbmdNZXRob2RUeXBlOjE1"
+    input: {addPostalCodeRules: [{end: "", start: "83-330"}], 
+      deletePostalCodeRules: [], 
+      inclusionType: EXCLUDE
+    }
+  ) {
+    errors {
+      field
+      code
+      message
+    }
+    shippingMethod {
+      postalCodeRules {
+        id
+        start
+        end
+        inclusionType
+      }
+    }
+    __typename
+  }
+}
+```
+
+### Product Exclusions
+
+You can exclude specific products from using certain shipping methods.
+
+- Use [`shippingPriceExcludeProducts`](/api-reference/shipping/mutations/shipping-price-exclude-products.mdx) 
+to add products to the exclusion list
+- Use [`shippingPriceRemoveProductFromExclude`](/api-reference/shipping/mutations/shipping-price-remove-product-from-exclude.mdx) 
+to remove products from the exclusion list
+
+Here is example for excluding products:
+```graphql {2-4}
+mutation {
+  shippingPriceExcludeProducts(
+    id: "U2hpcHBpbmdNZXRob2RUeXBlOjE1"
+    input: {products: ["UHJvZHVjdDoxNDQ="]}
+  ) {
+    errors {
+      field
+      code
+      message
+    }
+    shippingMethod {
+      excludedProducts {
+        totalCount
+        edges {
+          node {
+            slug
+          }
+        }
+      }
+    }
+  }
+}
+```
+### Delivery Time Estimates
+
+Configure delivery time estimates to help customers choose the right shipping method:
+
+```graphql
+{
+  "maximumDeliveryDays": 7,
+  "minimumDeliveryDays": 2
+}
+```
+
+### Tax Configuration
+
+You can assign a tax class to shipping methods. For more information, see the 
+[taxes guide](/developer/taxes.mdx#assigning-tax-rate-to-a-shipping-method).
+
+## Managing Shipping Methods
+
+### Updating
+
+Update existing shipping methods using:
+- [`shippingPriceUpdate`](/api-reference/shipping/mutations/shipping-price-update.mdx)
+ Use this mutation for general updates to the shipping method's name, delivery days, etc.
+- [`shippingMethodChannelListingUpdate`](/api-reference/shipping/mutations/shipping-method-channel-listing-update.mdx)
+ Use this mutation to adjust channel-specific settings like price and order total range.
+- [`shippingPriceTranslate`](/api-reference/miscellaneous/inputs/shipping-price-translation-input.mdx)
+ Use this mutation to provide translations for the shipping method's name and description in different languages.
+
+### Metadata Management
+
+To manage shipping rate metadata, use:
+- [`updateMetadata`](/api-reference/miscellaneous/mutations/update-metadata.mdx)
+- [`updatePrivateMetadata`](/api-reference/miscellaneous/mutations/update-private-metadata.mdx)
+
+### Deleting
+
+Remove shipping methods using:
+- [`shippingPriceDelete`](/api-reference/shipping/mutations/shipping-price-delete.mdx) for single deletion
+- [`shippingPriceBulkDelete`](/api-reference/shipping/mutations/shipping-price-bulk-delete.mdx) for bulk deletion
+
+## Webhooks
+
+Saleor provides the following webhooks for shipping methods:
```

Comment:
Maybe also worth to link general webhook page here?
Something like: "to familiar with webhook approach see [webhook chapter] " 🤔